.ci/build.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  python_versions : []
+
+steps:
+  - ${{ each py_version in parameters.py_versions }}:
+    - task: usePythonVersion@0
+      inputs:
+        versionSpec: '3.6'
+
+    - checkout: self
+      clean: true
+
+    - script: python3 -m pip install --user pyyaml
+      displayName: 'Install PyYaml'
+
+    - script: python3 jellyfin-kodi/.config/generate_xml.py ${{ py_version }}
+      displayName: 'Create ${{ py_version }} addon.xml'
+
+    - script: zip -r plugin.video.jellyfin-${{ py_version }}.zip jellyfin-kodi
+      displayName: 'Create ${{ py_version }} zip file'
+
+    - script: ls .
+      displayName: 'home dir 2'
+
+    - task: CopyFilesOverSSH@0
+      displayName: 'Upload to repo server'
+      inputs:
+        sshEndpoint: repository
+        sourceFolder: '${Agent.BuildDirectory}'
+        contents: 'plugin.video.jellyfin*'
+        targetFolder: '/srv/repository/incoming/kodi'
+
+    - task: SSH@0
+      displayName: 'Add to Kodi repo'
+      inputs:
+        sshEndpoint: repository
+        runOptions: 'inline'
```

Comment:
You can probably use the commands runOptions here, but if this works it works.
And this now runs for every build including for PRs, that might not be what we want.

Reply:
I don't have a repository set up in my account so I can't confirm it works, but I've updated it to more closely match what's in the server using `commands` instead, and added a condition for the tag